asp-core/src/main/java/de/jcup/asp/core/CryptoAccess.java

```diff
@@ -72,7 +75,7 @@ public String encrypt(String strToEncrypt) {
         try {
             Cipher cipher = Cipher.getInstance(SECURITY_ALGORITHM);
             cipher.init(Cipher.ENCRYPT_MODE, secretKey, new SecureRandom());
-            byte[] encryptedBytes = cipher.doFinal(strToEncrypt.getBytes("UTF-8"));
+            byte[] encryptedBytes = cipher.doFinal(escapeJava(strToEncrypt).getBytes("ISO-8859-1"));
             return Base64.getEncoder().encodeToString(encryptedBytes);
```

Comment:
If I read properly the javadoc of [encodeToString](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.Encoder.html#encodeToString-byte:A-). It should be possible to just call `new String(Base64.getEncoder().encode(encryptedBytes),StandardCharsets.UTF_8)`. I'll have a try and if it works I can remove the escapeJava workaround.